plaso/parsers/text_parser.py

```diff
@@ -582,6 +594,10 @@ class PyparsingMultiLineTextParser(PyparsingSingleLineTextParser):
 
   BUFFER_SIZE = 2048
 
+  # The maximum number of consecutive lines that don't match known line
+  # structures to encounter before aborting parsing.
+  MAXIMUM_CONSECUTIVE_LINE_FAILURES = 20
```

Comment:
Since PyparsingMultiLineTextParser inherits from PyparsingSingleLineTextParser, can you remove it here?

Reply:
That's a good point, I can indeed.